src/loader/timescale_bgw_internal.h

```diff
@@ -0,0 +1,25 @@
+#ifndef TIMESCALE_BGW_INTERNAL_H
+#define TIMESCALE_BGW_INTERNAL_H
+
+#include <postgres.h>
+
+/* Interface internal to the loader */
+/* would like feedback on whether to do this or to make (some of) these static and include timescale_bgw.c in the loader*/
+
+extern void timescale_bgw_shmem_init(void);
+extern void timescale_bgw_shmem_startup(void);
+
+extern void timescale_bgw_on_db_drop(Oid dropped_db);
+extern void timescale_bgw_on_db_create(Oid created_db);
+
+extern void timescale_bgw_register_cluster_launcher(void);
```

Comment:
timescale_bgw_cluster_launcher_register()

Reply:
I don't know why we are putting verbs at the end, it's rather unintuitive and if I had a register of cluster launchers, I would call it a timescale_bgw_cluster_launcher_register and then everyone would be very confused. But if that is protocol...